modules/packages/LAPACK.chpl

```diff
@@ -19092,6 +19092,7 @@ Wrapped procedure of LAPACKE_sbdsdc for the type real(32).
  */
 inline proc bdsdc(matrix_order : lapack_memory_order, uplo : string, compq : string, n : c_int, d : [] real(32), e : [] real(32), u : [] real(32), vt : [] real(32), q : [] real(32), iq : [] c_int): c_int{
   require header;
+  import LAPACK.ClassicLAPACK;
```

Comment:
There were enough changes in this file to warrant Github hiding the contents initially, I would strongly consider having a private import outside of the scope of the functions instead